debian/changelog

```diff
@@ -1,3 +1,26 @@
+opx-nas-ndi (4.8.0) unstable; urgency=medium
+
+  * Feature: VxLAN Support
+  * Bugfix: Fix nas mac callback event
+  * Bugfix: Fix ip address family for mac events
+  * Bugfix: Check bridgeport ID before flushing
+  * Update: Add Ipv6 Extended route configuration handle
+  * Update: Add handle_profile_map() back to the code
+  * Update: Add split horizon attribute set for subport
+  * Update: Ignore mac flush event from SAI
+  * Update: Add support for virtual rif attribute as part of RIF configuration
+  * Update: Add ipv4 and ipv6 interface statistics attributes
+  * Update: Add support for setting multicast lookup key in VLAN
+  * Update: Add support port vlan subport flush
+  * Update: Clear stat implementation for bridge,bridge port and tunnel
```

Comment:
Need a space between "bridge," and "bridge port" + oxford comma after "bridge port"